android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -417,49 +417,71 @@ public boolean showBiometricPrompt() {
     }
 
     @Override
-    public void showBiometricPrompt(boolean show) {
-        mShowBiometricPrompt = show;
+    public void enableBiometricPrompt() {
```

Comment:
Renamed `showBiometricPrompt` to `enableBiometricPrompt`.
Removed boolean from signature as always `true`.